dashboard/test/models/foorm/library_question_test.rb

```diff
@@ -79,4 +79,19 @@ class Foorm::LibraryQuestionTest < ActiveSupport::TestCase
     }"
     assert_empty library_question.published_forms_appeared_in
   end
+
+  test 'library question JSON cannot be updated with question name different than what is in' do
```

Comment:
Did you mean, "..different than what is in the database"? I'm also curious, is it possible to update a question name? (sorry for the basic questions, I don't know much about this space)

Reply:
Yikes, yes typo thanks!

Reply:
Re: can a library question name change...we don't provide UI to do this, and I kind of think we should not allow it (at least for questions that have been used in Forms). The idea being that we wouldn't want to have orphaned submissions to a question (where we've logged the question name as part of the submission, but the question no longer exists).